src/pages/press.md

```diff
@@ -27,129 +27,61 @@ news:
   title: In the News
   body: Featured Coverage
   cards:
-    - title: How Millennial Finance Will Make The Legacy Banking System Irrelevant
-      source: IBTimes
-      excerpt: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit cursus sed feugiat iaculis dictumst.
-      date: "2020-09-22T13:00:00.000Z"
+    - title: Keep Network unveils v2 specs for tBTC protocol
+      source: Cointelegraph
+      excerpt: The second iteration of tBTC is expected to require stakers to only lock up Keep rather than both Keep and ETH, alongside introducing changes to its wallet-generation mechanism. The protocol allows users to tokenize their Bitcoin for use on the Ethereum network.
+      date: "4/12/2021"
       icon:
         image: /images/home/keep-blog.png
-        alt: Forbes
-      url: https://www.coindesk.com/thesis-bitcoin-token-tbtc-relaunches-following-buggy-debut
-    - title: Bitcoin-Backed Ethereum Token tBTC Launches—for the Second Time
-      source: Decrypt
-      excerpt: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elit cursus sed feugiat iaculis dictumst.
-      date: "2020-09-22T13:00:00.000Z"
+        alt: Cointelegraph
+      url: https://cointelegraph.com/news/keep-network-unveils-v2-specs-for-tbtc-protocol
+    - title: "‘Keanu’ Explained: What It Means to Merge Two Ethereum Projects"
+      source: CoinDesk
+      excerpt: "Two encryption projects, Keep and NuCypher, both running on the Ethereum blockchain, have begun discussing what they are calling a “hard merge,” codename: Keanu. Crucially, this would be a merger of their protocols’ functions and communities, not of their companies."
+      date: "3/11/2021"
       icon:
         image: /images/home/keep-blog.png
-        alt: Forbes
-      url: https://www.coindesk.com/thesis-bitcoin-token-tbtc-relaunches-following-buggy-debut
+        alt: CoinDesk
+      url: https://www.coindesk.com/keanu-explained-nucypher-keep-merger-ethereum
+    - title: Keep Network, NuCypher teams pitch protocol merger on the Ethereum blockchain
+      source: The Block
+      excerpt: Development teams for Keep Protocol and NuCypher are proposing an on-chain merger in a network first for Ethereum.
+      date: "3/8/2021"
+      icon:
+        image: /images/home/keep-blog.png
+        alt: The Block
+      url: https://www.theblockcrypto.com/post/97634/keep-protocol-nucypher-ethereum-merger
 press_items_section:
   title: Latest Coverage
   press_items:
-    - date: "2020-09-22T19:25:00.000Z"
```

Comment:
Hey @FrozenIce0617 - please keep all the OLD articles under "Latest Coverage". The articles in the Google doc are **in addition to** the existing articles, not replacing them.

Reply:
Oh, I see.